src/main/java/org/cdlib/ill/report/excel/ScheduleCController.java

```diff
@@ -0,0 +1,283 @@
+  private ScheduleCRepository scheduleCRepo;
+  @Autowired
+  private SpVdxBorrowingOCLCRepository spVdxBorrowingOCLCRepo;
+
+  private static final ClassPathResource TEMPLATE = new ClassPathResource("org/cdlib/ill/report/ScheduleC_Template.xlsx");
+
+  private static final Supplier<IllegalArgumentException> NO_SUCH_CAMPUS = () -> {
+    return new NoSuchCampusException("Schedule C is only available for individual campuses. Please choose a campus.");
+  };
+
+  @RequestMapping(
+      value = "{campusCode}/schedule_c.xlsx",
+      produces = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
+  public void getScheduleC(
+      @PathVariable("campusCode") String campusCode,
+      OutputStream clientDownload,
+      @RequestParam(required = false, name = "startDate", defaultValue = "1900-01-01") @DateTimeFormat(pattern = "yyyy-MM-dd") LocalDate startDate,
+      @RequestParam(required = false, name = "endDate", defaultValue = "2100-01-01") @DateTimeFormat(pattern = "yyyy-MM-dd") LocalDate endDate) throws IOException {
+
+    // A specific campus is required.
+    VdxCampus campus = VdxCampus.fromCode(campusCode).orElseThrow(NO_SUCH_CAMPUS);
+
+    try (XSSFWorkbook wb = new XSSFWorkbook(TEMPLATE.getInputStream())) {
+      buildScheduleCSheet(wb, campus, startDate, endDate);
+      buildUCBorrowingSheets(wb, campus, startDate, endDate);
+      buildOCLCBorrowingSheets(wb, campus, startDate, endDate);
+      buildLendingSheets(wb, campus, startDate, endDate);
+      wb.write(clientDownload);
+    }
+  }
+
+  // The order that the campuses appear in the XLSX template file.
+  private static final List<VdxCampus> CAMPUS_ORDER = Arrays.asList(
+      VdxCampus.Berkeley,
+      VdxCampus.Davis,
+      VdxCampus.Irvine,
+      VdxCampus.LosAngeles,
+      VdxCampus.Merced,
+      VdxCampus.Riverside,
+      VdxCampus.SanDiego,
+      VdxCampus.SanFrancisco,
+      VdxCampus.SantaBarbara,
+      VdxCampus.SantaCruz,
+      VdxCampus.NorthernRegionalLibraryFacility,
+      VdxCampus.SouthernRegionalLibraryFacility
+  );
+
+  private static final Set<VdxCampus> RLFS = EnumSet.of(VdxCampus.NorthernRegionalLibraryFacility, VdxCampus.SouthernRegionalLibraryFacility);
+
+  private void buildScheduleCSheet(XSSFWorkbook wb, VdxCampus campus, LocalDate startDate, LocalDate endDate) {
+    final XSSFSheet scheduleCSheet = wb.getSheet("Schedule C");
+    scheduleCSheet.createRow(0).createCell(0).setCellValue("UC Libraries Statistics (" + startDate.toString() + " to " + endDate.toString() + ")");
+    scheduleCSheet.getRow(2).createCell(1).setCellValue(campus.getDescription());
+
+    // Assumption that the "UC Libraries" row is grayed out.
+    XSSFCellStyle disabledStyle = scheduleCSheet.getRow(6).getCell(1).getCellStyle();
+
+    // Clear the home campus values on "Schedule C"
+    XSSFRow disabledRow1 = scheduleCSheet.getRow(7 + CAMPUS_ORDER.indexOf(campus));
+    for (short index = 1; index < disabledRow1.getLastCellNum(); index++) {
```

Comment:
Seems like for loop:  for (short index = 2 ... index +=2 ) { 
would work - instead of going through all the values and checking for index %2.